realtime_tools/include/realtime_tools/realtime_box.hpp

```diff
@@ -33,278 +33,17 @@
 #ifndef REALTIME_TOOLS__REALTIME_BOX_HPP_
 #define REALTIME_TOOLS__REALTIME_BOX_HPP_
 
-#include <functional>
-#include <initializer_list>
-#include <mutex>
-#include <optional>
-#include <utility>
+#include "realtime_tools/realtime_threadsafe_box.hpp"
 
-#include <rcpputils/pointer_traits.hpp>
+// Deprecation notice
+#pragma message( \
+  "'RealtimeBox' is deprecated. Please update your code to use 'realtime_threadsafe_box.hpp' header, and class name 'RealtimeThreadsafeBox' instead.")  //NOLINT
```

Comment:
```suggestion
  "'RealtimeBox' is deprecated. Please update your code to use 'realtime_threadsafe_box.hpp' header, and class name 'RealtimeThreadSafeBox' instead.")  //NOLINT
```

Can we call with S capital?